.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,37 @@
+################################################################################
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+name: Nightly
+on:
+  schedule:
+    # Run at 2:00 daily.
+    - cron: "0 2 * * *"
```

Comment:
This CRON is scheduled in UTC time zone, so it's actually 10:00 in UTC+8. I suggest to run at 20:00 UTC, because it's the lowest commit traffic according to the [ossinsight](https://ossinsight.io/analyze/apache/fluss#commits)

<img width="1362" height="996" alt="image" src="https://github.com/user-attachments/assets/7f042353-d4c1-4ba1-843a-e8764301dc53" />